src/components/Room.tsx

```diff
@@ -22,6 +23,8 @@
 	const [usersInRoom, setUsersInRoom] = useState<number>(0);
 	const [isTimerPaused, setIsTimerPaused] = useState<boolean>(false);
 	const [userListInRoom, setUserListInRoom] = useState<string[]>([]);
+	// TODO: setBreak should set the state after receiving response from the server
+	const [isBreak, setBreak] = useState(false);
```

Comment:
Is this okay? I've left the setBreak variable there since it would be in-line with the code design even though it's not being used, yet.

Also the take a break button is only emitting breakTimer events to server now since the state isBreak is not being changed.

Reply:
Yes both are fine. 

Don't forget to add the type to the set state.